networks/mainnet/rpcs.md

```diff
@@ -4,48 +4,170 @@ description: Public RPC endpoints are available for the Filecoin mainnet.
 
 # RPCs
 
-{% hint style="info" %}
-[Chainlist](https://chainlist.org/?search=filecoin&testnets=true) contains a dynamically updated list of available Filecoin RPCs. [Find out more at chainlist.org's Filecoin listings](https://chainlist.org/?search=filecoin&testnets=true).
-{% endhint %}
-
-These endpoints are limited to [read-only Filecoin JSON RPC API calls](../../reference/json-rpc/) and [`MPoolPush`](../../reference/json-rpc/mpool.md) for sending messages that have already been signed.
-
-## [Ankr](https://ankr.com)
-
-* HTTPS: `https://rpc.ankr.com/filecoin`
-* [Supported Filecoin API methods](https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin)
-
-## [ChainupCloud](https://cloud.chainup.com)
-
-* HTTPS: `https://filecoin.chainup.net/rpc/v1`
-* WebSocket: `wss://filecoin.chainup.net/rpc/v1`
-* [ChainupCloud documentation](https://docs.chainupcloud.com/blockchain-api/filecoin/public-apis)
-
-## [Glif](https://api.node.glif.io)
-
-Please note that publicly available hosted endpoints **only guarantee 2000 of the latest blocks.**
-
-* HTTPS: `https://api.node.glif.io/rpc/v1`
-* WebSocket: `wss://wss.node.glif.io/apigw/lotus/rpc/v1`
-*   Lotus lite-node command:
+[Chainlist](https://chainlist.org/?search=filecoin&testnets=true) provides a dynamically updated list of [available Filecoin - Mainnet RPCs](https://chainlist.org/?search=filecoin&testnets=true).
+
+These endpoints are limited to [read-only Filecoin JSON RPC API calls](../../reference/json-rpc/) including Filecoin Eth RPC methods and [`MPoolPush`](../../reference/json-rpc/mpool.md) for sending already signed messages.
+
+{% hint style="info" %} Please note that most publicly hosted endpoints **only guarantee recent state, i.e. 2000 of the latest blocks (last 16.67 hours).** To request an archival node you can contact a provider below. {% endhint %}
+
+
+<table>
```

Comment:
A few thoughts?
1. Add headers?
2. Instead of doing "rowspans" do one row per provider?
3. Do the table in markdown in rather than html so easier to read in non-rendered mode?

Reply:
@BigLep - I'm just going to play favorites for now given current perf issues + historical responsiveness of the teams + how closely they work with FilOz to save devs time, once RPC stabilizes more generally we can revise.

Reply:
Let me move these suggestions to an issue here so we can get this merged sooner then iterate:

https://github.com/filecoin-project/filecoin-docs/issues/2320